test/browser/render.js

```diff
@@ -185,6 +185,27 @@ describe('render()', () => {
 			.and.matches(/position\s*:\s*relative\s*/);
 	});
 
+	it('string style should call setAttribute', () => {
+		const style = 'background-color: red;';
+		const spy = sinon.spy(Element.prototype, 'setAttribute');
+		render(<div style={style} />, scratch);
+		
+		expect(spy.firstCall.args).to
+			.deep.equal(['style', style]);
+		expect(scratch.childNodes[0]).to.have.deep.property('style.cssText')
+			.that.equals(style);
+		spy.restore();
+	});
+
+	it('object style prop should not call setAttribute', () => {
+		const style = { backgroundColor: 'papayawhip' };
+		const spy = sinon.spy(Element.prototype, 'setAttribute');
+		render(<div style={style} />, scratch);
+		
+		expect(spy).not.to.have.been.called;
```

Comment:
Do you mean `expect(spy).not.to.have.been.called()` ?

Reply:
the type indicates that it is a property, not a function:

```
        interface Assertion {
            /**
             * true if the spy was called at least once.
             */
            called: Assertion;
```

is this incorrect?

Reply:
changing to a function call throws an error. called is meant to be accessed as a property.

Reply:
Based on the example here `https://www.chaijs.com/plugins/chai-spies/` it is a func. Maybe you are right. I didn't know it works that way.

Reply:
the docs may be out of date. the types are directly from the repo